Generate.py

```diff
@@ -61,6 +61,9 @@ def _handle_part(cls, part: str, base: PlandoSettings) -> PlandoSettings:
         else:
             return base | part
 
+    def __str__(self) -> str:
+        return f"{[part.name for part in self]}"
```

Comment:
Have you tried running that? From my testing the instance is not iterable, so that shouldn't work - which means we'll have to construct it ourselves with `&` or `in`.
(I feel like Flags would be more appropriate here than IntFlags, but the automatic naming is still not nice, so we wouldn't really win anything)
```suggestion
    def __str__(self) -> str:
        if self.value:
            return ', '.join((flag.name for flag in PlandoSetting if self.value & flag.value))
        return 'Off'
```

Reply:
I think I did that originally and it failed so I did class method which worked but forgot lol